contracts/KSFairLaunchV3.sol

```diff
@@ -66,7 +66,10 @@ contract KSFairLaunchV3 is IKSFairLaunchV3, KyberSwapRole, ReentrancyGuard {
     }
 
     uint256 multiplierTo;
-    for (uint256 i = 0; i < rewardAmounts.length; i++) {
+    for (uint256 i = 0; i < rewardAmounts.length; ) {
```

Comment:
just `uint256 i` is enough, it's 0 by default

Reply:
fixed